index.js

```diff
@@ -14,6 +14,19 @@ module.exports.pitch = function (request) {
 
 	var options = loaderUtils.getOptions(this) || {};
 
+	var attrs = {};
```

Comment:
Use the `request` or `this.resourcePath` and shorten it instead of interpolating, we already have the`url` here  

```js
var attrs = {
   'data-url': request || this.resourcePath
}
```